frontend/tablet/package.json

```diff
@@ -16,6 +16,7 @@
     "@react-navigation/native": "^6.0.6",
     "@react-navigation/stack": "^6.0.11",
     "@types/react-fontawesome": "^1.6.5",
+    "moment": "^2.29.1",
```

Comment:
![image](https://user-images.githubusercontent.com/43814121/149664760-7602efcb-1017-4260-8ab6-e480c8b9449b.png)

It is no longer recommended to use this library (by the devs of this library).